pkg/didcomm/common/service/service.go

```diff
@@ -20,6 +27,14 @@ type DIDComm interface {
 	Event
 }
 
+// Common keeps helpful or common fields which can be defined in payload,
+// this structure was created to avoid json.Unmarshal twice
+type Common struct {
```

Comment:
This looks a bit like header fields that we are extracting from the message.

I don’t think that common is descriptive.

Reply:
I agree. Done!